build.sbt

```diff
@@ -1,10 +1,11 @@
 import akka.grpc.Dependencies
 import akka.grpc.Dependencies.Versions.{ scala212, scala213 }
-import akka.grpc.ProjectExtensions._
+import akka.grpc.ProjectExtensions.*
 import akka.grpc.build.ReflectiveCodeGen
-import com.typesafe.tools.mima.core._
+import com.typesafe.tools.mima.core.*
```

Comment:
?

Reply:
Huh, IntelliJ what on earth are you up to?

Reply:
Also, why was it there to start with? Removed it now.